src/Components/Buttons/FilterButton/FilterButton.js

```diff
@@ -0,0 +1,44 @@
+// Import Components
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// Import Stylings
+import './FilterButton.css';
+
+// Import Icons
```

Comment:
Import icons? Only a comment?

Reply:
Thank you! I just removed this unnecessary comments!